app/views/merge_requests/confirm_telephone_number.html.erb

```diff
@@ -10,19 +10,18 @@
   <div class="govuk-grid-column-two-thirds-from-desktop">
     <% if @merge_request.absorbing_organisation.phone.present? %>
       <p class="govuk-body">Confirm the telephone number on file, or enter a new one.</p>
+      <div class="govuk-inset-text">
+        <%= @merge_request.absorbing_organisation.phone %>
+      </div>
     <% end %>
 
-    <div class="govuk-inset-text">
-      <%= @merge_request.absorbing_organisation.phone %>
-    </div>
-
     <%= form_with model: @merge_request, url: merge_request_path, method: :patch do |f| %>
       <%= f.govuk_error_summary %>
```

Comment:
I just realised that the error should probably be above the question header, it's wrong on all pages 😭